src/main/java/co/novu/common/rest/RestHandler.java

```diff
@@ -61,4 +68,16 @@ public <T, R> R extractResponse(Response<T> response, R body) throws NovuNetwork
             throw new NovuNetworkException(response.errorBody() != null ? response.errorBody().string() : "Error connecting to Novu API");
         }
     }
+
+    private String loadSdkVersionFromPom(){
+        try {
+            MavenXpp3Reader reader = new MavenXpp3Reader();
+            Model model = reader.read(
+                new InputStreamReader(Objects.requireNonNull(this.getClass().getResourceAsStream("/META-INF/maven/co.novu/novu-java/pom.xml"))));
+            return model.getVersion();
+        } catch (Exception e) {
+            e.printStackTrace();
```

Comment:
Annotate this class with Slf4j and log the exception

Reply:
Added the suggested changes